portfolio/src/main/webapp/script.js

```diff
@@ -111,6 +58,36 @@ function createListElement(text) {
   return liElement;
 }
 
+function createPostElement(postJson) {
+  const post = document.createElement("div");
+
+  const postHeader = document.createElement("div");
+  post.appendChild(postHeader);
+  const usernameDisplay = document.createElement("p");
+  const timestampDisplay = document.createElement("p");
+  postHeader.appendChild(usernameDisplay);
+  postHeader.appendChild(timestampDisplay);
+
+  usernameDisplay.innerText = "user";
```

Comment:
Should this be hardcoded or are you planning to get this information from the server too?

Reply:
Will add in future commit! Will leave unresolved for now.

Reply:
Resolved in later commit.